### DISCUSSION TITLE
Adding local storage

### DISCUSSION THREAD

**Discussion by JC-os** [audioling/audioling#3] [Ideas]
Good morning everybody and greetings Jeff for all your work and the beautiful software you offered to the world.
As you asked for some suggestions, I would like to ask you if it would be possible to have a local storage access to folder on the hd computer.

I actually use itunes for my local folders (just to be eco-respectful limiting the internet usage) and feishin for my full library (I connect to a navidrome docker container on a personal server).

But I would appreciate to use feishin/audioling only and drop itunes, so I would like to ask (I have no developer skills) if it would be possible to add an access to local folders or a download option in feishin/audioling (offline mode).

Thank you again and happy self-hosting to everyone!

**Comment by jeffvli** [comment]
Local storage / files support -- most likely not. My intention is to allow the media server (in your case Navidrome) to do all the heavy lifting with folder scanning, tag reading, etc to get all the metadata. Adding a local file scanner would be out of the scope of what I want to implement.

On the other hand, downloading and offline playback **is planned** to be implemented.

**Comment by JC-os** [comment]
Thank you Jeff, I understand your scope.
Downloading and offline playback will be enough as offline playback is the real core of my suggestion.
At that point I close the thread and I wish you a good work with some great contributors!
Thank you very much again.
